setup.py

```diff
@@ -7,9 +7,9 @@
     install_requires=[
         "sqlalchemy",
         "psycopg2-binary",
-        "pandas<1.1",
```

Comment:
I am still having issues with the key=abs as you mentioned in the write up. Do we have to specify the version of pandas here?